lib/minimart/mirror/inventory_builder.rb

```diff
@@ -33,10 +34,16 @@ def build!
       # precedence over information found elsewhere.
       def install_cookbooks_with_explicit_location
         inventory_requirements.each_with_explicit_location do |requirement|
-          requirement.fetch_cookbook do |cookbook|
-            validate_cookbook_against_local_store(cookbook, requirement)
-            add_artifact_to_graph(cookbook)
-            add_cookbook_to_local_store(cookbook.path, requirement.to_hash)
+          begin
```

Comment:
I don't feel good about wrapping this entire block with a begin that doesn't do any actual handling. Can we isolate which specific call would generate the error and focus on saving that with some handling in the rescue?